src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![doc(html_root_url = "https://docs.rs/semver-parser/0.7.0")]
```

Comment:
is there a reason to link to a specific version here?

Reply:
Looks like it's no longer needed since... two days ago :smile: https://github.com/rust-lang-nursery/api-guidelines/commit/5c457dfa5340fd3ba2ede0ca6d5896c48d1e7d49

Reply:
Nice! Let's do that then; much cleaner this way 😄

Reply:
We should close the this issue then? And remove other html_root_url?

Reply:
@opilar see the discussion here https://github.com/mgeisler/version-sync/pull/27